dataflow-migrate-schedules/README.adoc

```diff
@@ -102,24 +101,75 @@ To enable this feature use the `runMigration.sh` script as follows.
 ./scripts/runMigration.sh --spring.cloud.task.single-instance-enabled=true
 ```
 
-=== Configuring Your Deployer Properties
-The following deployer properties will affect all schedules to be migrated.
-If a property is not set then the default will be used.
+=== Previously Pushed Apps
+The Cloud Schedule Migration app deletes SchedulerTaskLauncher and its associated schedule.
 
-==== Deployer properties to be applied to all migrated schedules:
-* healthCheckTimeout
-* apiTimeout
-* statusTimeout
-* stagingTimeout
-* startupTimeout
-* maximumConcurrentTasks
-* javaOpts
+NOTE: If a single task definition has more than one schedule, the task definition
+droplet created for first migrated schedule will be used for all scheduled tasks of that task definition.
 
-NOTE: Descriptions of these properties can be found : https://github.com/cppwfs/spring-cloud-dataflow-samples/blob/SCDF-121/dataflow-migrate-schedules/src/main/java/io/spring/migrateschedule/service/MigrateProperties.java[here]
+== Running The Project For Kubernetes
 
-=== Supported Databases
-The database supported are enumerated https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-local-rdbms[here].
+=== Prerequisites
 
-=== Previously Pushed Apps
-The Cloud Schedule Migration app does not delete previously scheduled applications.
-If these apps are no longer needed it is up to the user to delete them.
+* Spring Cloud Data Flow 2.3.2.RELEASE, 2.4.3.RELEASE, and 2.5.0.RELEASE must be installed and running prior to launching the migration app.
+* Spring Cloud Data Flow must be using the `exec` entry point for scheduling apps.
+
+=== Launching your migration
+1) Establish the following environment variables:
+```
+export spring_profiles_active=kubernetes
+export spring_autoconfigure_exclude=org.springframework.cloud.deployer.spi.cloudfoundry.CloudFoundryDeployerAutoConfiguration
+export spring_datasource_url=<The Spring Cloud Data Flow database URL to be used by the migration tool>
+export spring_datasource_username=<The Spring Cloud Data Flow database username to be used by the migration tool>
+export spring_datasource_password=<The Spring Cloud Data Flow database password to be used by the migration tool>
+export spring_datasource_driverClassName=<The Spring Cloud Data Flow database driverClassName to be used by the migration tool>
+export KUBERNETES_NAMESPACE=< The namespace for the scheduled apps >
+export dbDriverClassName=<The Spring Cloud Data Flow database driverClassName to be used by the migrated tasks>
```

Comment:
these seem duplicated given the exist with spring_ prefix above?

Reply:
understood why now re: dup -- the spring_datasource_url could be anything (ie: mysql port forwarded to localhost, etc) where in dbUrl you want the actual address of the DB

Reply:
After the migration, I see that the `spring_` properties are what would become the datasource properties for the scheduled tasks. Hence, using the `localhost` as the `spring_datasource_url` wouldn't work and this has to be the actual DB host address.